pages/linux/efibootbgr.md

```diff
@@ -0,0 +1,23 @@
+# efibootmgr
+
+> Manipulate the UEFI Boot Manager (the Bootoptions).
+
+- List the current settings:
+
+`efibootmgr `(`-v` for Filepaths)
```

Comment:
The above command doesn't work for me, it needs another `, and even if I add it I the next traceback (syntax error) before the settings of my boot manager:

```
bash: command substitution: línea 2: error sintáctico: no se esperaba el final del fichero
bash: command substitution: línea 1: error sintáctico cerca del elemento inesperado `)'
bash: command substitution: línea 1: ` for Filepaths)'
```

The suggestion I do below works nice for me. It might be useful to have the feedback of another reviewers to test this.
```suggestion
`efibootmgr -v for Filepaths`
```

Reply:
This one [has been reported](https://github.com/tldr-pages/tldr/pull/3788#issuecomment-577631625) tldr-bot.